test/Android/Breakpoint/TestScript.xml

```diff
@@ -111,7 +111,7 @@
     <Event name="IDebugThreadCreateEvent2" expected="False" />
     <Event name="IDebugOutputStringEvent2" expected="False" />
     <Event name="IDebugBreakpointBoundEvent2" expected="True">
-      <BoundBreakpoint>24,0 to 24,0 in [Unknown]!break_test_1()</BoundBreakpoint>
+      <BoundBreakpoint RegEx="24,0 to 24,0 in (\[Unknown\]|libbreakpoint.so)!break_test_1()"></BoundBreakpoint>
     </Event>
```

Comment:
Might want to use '^<your-pattern>$' so that you make sure the pattern only matches the whole string.
Might want to add your comment as a comment to the code.